backend/queue_cli/utils/display.py

```
@@ -0,0 +1,179 @@
+    print_info(f"State: [bold]{state}[/bold]")
+
+
+def print_next_steps(task_id: str, queue: str) -> None:
+    """Print next steps for the user."""
+    console.print()
+    console.print("[bold]🔍 Next Steps:[/bold]")
+    console.print(f"  • Check task status: [cyan]uv run python queue_cli.py status {task_id}[/cyan]")
+    console.print(
+        f"  • View queue: [cyan]uv run python queue_cli.py queues --name {queue}[/cyan]"
+    )
+    console.print(f"  • View workers: [cyan]uv run python queue_cli.py workers[/cyan]")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)